### PR TITLE
Shiftry Release Code

### PR DESCRIPTION
**Pull request by aayan636**

### CODE REVIEW

File: tools/SeeDot/seedot/compiler/TF/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ShikharJ) [inline tools/SeeDot/seedot/compiler/TF/__init__.py:1]: Do we need this file? I think it's better to remove this.

File: tools/SeeDot/seedot/compiler/input/profile.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+-49.318436, 298.254303
+0.015436, 49.318436
```
Comment (ShikharJ) [inline tools/SeeDot/seedot/compiler/input/profile.txt:2]: Do we need this file as well?

File: tools/SeeDot/seedot/arduino/compileConfig.h [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+// The datatype of the fixed-point representation is specified below.
+#define INT16
+#define XFLOAT
```
Comment (ShikharJ) [inline tools/SeeDot/seedot/arduino/compileConfig.h:6]: Ideally, these should be compile time flags, no? I don't see why these need to be defined in a separate file. I think it's better to document these instead.

**Comment by ShikharJ** [comment]
@harsha-simhadri All the latest changes have been pushed to this PR, and from hereon, this code should be referred for the latest release for SeeDot. Regarding the code with the hacks for Face Detection models, I'll open a separate branch and push a slightly modified version of this code there for use in our Jupyter Notebook.

**Comment by harsha-simhadri** [comment]
@aayan636  @ShikharJ shall we close this PR since you are working on the SeeDOT-Face-Detection branch for this PR?

**Comment by ShikharJ** [comment]
I don't think that'd be a good idea, since this PR has diverged from the SeeDOT-Face-Detection PR, in the sense that it doesn't contain any hacky codes. The latter is geared more towards the Medium post, and as such has files which don't need to be committed to master branch.

**Comment by ShikharJ** [comment]
@harsha-simhadri Can we get this merged?

**Review by ShikharJ — APPROVED** [review]
This PR is complete now.

**Comment by ShikharJ** [comment]
@harsha-simhadri @aayan636 @rahsharmar I've done extensive testing and eyeballed the README instructions. I'm merging this in for now. Through our experiments on MinUn, we've discovered more improvements to the pipeline. Those will be pushed in a separate PR.

**Pull request merged**